model/Vocabulary.php

```diff
@@ -244,7 +244,7 @@ public function getDefaultConceptScheme()
     /**
      * Returns the main Concept Scheme of that Vocabulary, or null if not set.
      * @param string $defaultConceptSchemeURI default concept scheme URI
-     * @return \EasyRdf\Graph|mixed
+     * @return \EasyRdf\Graph|\EasyRdf\Sparql\Result
```

Comment:
I don't think this is correct. The return type (Graph vs Result) depends on the SPARQL query type: Graph for CONSTRUCT, Result for SELECT (and ASK). The query type is hardcoded so the type is already known within Skosmos code - but of course, code inspection tools cannot know the intricacies of SPARQL queries, so some kind of explicit PHP typecheck may be necessary to keep them happy. But I don't think this kind of "it could be either" annotation makes sense. Same applies for the similar changes you did to GenericSparql phpdocs.

Reply:
Should we use `Result` here then? I will look at the other change and use the value that we expect too if that's better?

Reply:
Dropped commit with interactive rebase, push-force'd.

Reply:
Okay great! I'm being careful here by not saying anything about whether to fix this or not, just wanted to point out that the change was not necessarily an improvement :)